aws_sagemaker_studio/sagemaker_data_prep_assistant/data_prep_widget.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>Use present tense for the whole notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3725/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>